### ISSUE TITLE
Not including a developer token makes it hard for people to contribute

### ISSUE THREAD

**Issue by peterjosling** [zachomedia/apple-music-webplayer#41]
Right now if someone wants to run this project locally, they need to either have their own Apple developer account and generate their own token, or pull up the compiled source from the production build on music.zacharyseguin.ca and find your token there.

The token was designed to be shipped in client-side code, so I don't see any huge concerns with publishing it in the repository. If someone wants to abuse the API with your token they can scrape the token from the built code anyway, so not publishing it here doesn't really give you much additional protection.

(Similarly with the Sentry plugin - right now you have to either delete the privateConfig import or create a new file before it will build)

**Comment by agg23** [comment]
I agree entirely. There is no real concept of a secret in the JavaScript world.

Though I haven't yet gotten around to making my intended contribution to this project, when I was first exploring the possibility, I was able to extract several tokens from other Github repos, due to either carelessness or recognition that the tokens cannot truly be secured in a production scenario.

**Comment by zachomedia** [comment]
I don't want to push the token into the public code at this point since I'm not sure if there are implications of doing so, especially with regards to the Apple Developer program terms.

**Comment by Kail0** [comment]
I was wondering the same thing. 

But I think it's not possible to hide it due to nature of how JWT works.

![image](https://user-images.githubusercontent.com/2068046/52331605-e4470a80-29f8-11e9-98ed-9f1b881578b6.png)
